build.gradle

```diff
@@ -12,6 +12,7 @@ repositories {
 }
 
 dependencies {
+    compile 'commons-cli:commons-cli:1.4'
```

Comment:
Замените, пожалуйста, `scope` `compile` на `implementation`, так как `compile`. 

**Выдержка из документации:**
> The compile configuration still exists but should not be used as it will not offer the guarantees that the api and implementation configurations provide.